### ISSUE TITLE
Traefik fails to obtain ClouDNS wildcard certificate

### ISSUE THREAD

**Issue by steve-todorov** [containous/traefik#4772]
### Do you want to request a *feature* or report a *bug*?

Bug (possibly related to #4665)

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

I have configured traefik to use LetsEncrypt with wildcard domains using the ClouDNS provider. 

### What did you expect to see?

Successful issue of a wildcard certificate

### What did you see instead?

```
traefik    | time="2019-04-15T17:01:53Z" level=info msg="legolog: [INFO] [*.domain.com] acme: Cleaning DNS-01 challenge"
traefik    | time="2019-04-15T17:01:54Z" level=info msg="legolog: [WARN] [*.domain.com] acme: error cleaning up: ClouDNS: TXT record unmarshaling error: json: cannot unmarshal array into Go value of type internal.TXTRecords "
traefik    | time="2019-04-15T17:01:54Z" level=error msg="Error obtaining certificate retrying in 618.499901ms"
```

I also monitored the dashboard in cloudns.net while Traefik was trying to update the DNS records. During this time the domain did not get updated. There was no change in the DNS records what so ever. 

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.7.10
Codename:     maroilles
Go version:   go1.11.6
Built:        2019-03-29_12:20:34PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
debug = false
logLevel = "INFO"
defaultEntryPoints = ["https", "http"]
[entryPoints]
  [entryPoints.http]
  address = ":80"
  [entryPoints.https]
    address = ":443"
    [entryPoints.https.tls]
      minVersion = "VersionTLS13"
      cipherSuites = ["TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256","TLS_RSA_WITH_AES_256_GCM_SHA384"]
[acme]
email = "abd@def.gh"
storage = "/acme.json"
entryPoint = "https"
acmeLogging = true
# PRODUCTION URL
caServer = "https://acme-v02.api.letsencrypt.org/directory"
  [acme.dnsChallenge]
    provider = "cloudns"
    delayBeforeCheck = 120
[[acme.domains]]
    main = "*.domain.com"
    sans = ["domain.com"]
```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

[traefik.log](https://github.com/containous/traefik/files/3081313/traefik.log)

**Comment by ldez** [comment]
@steve-todorov If I produce a custom Docker image (at first, I'll just add some log), are you able to quickly test this image?

**Comment by steve-todorov** [comment]
With the help of @ldez we were able to narrow down the issue to the `TTL` field for the DNS records. According to the [ClouDNS API docs](https://www.cloudns.net/wiki/article/58/) you can only use specific values for the `TTL` field. Unfortunately some of the requests to the API were silently failing because of the incorrect `TTL` value and thus there were no TXT records added to the DNS.

The workaround for `v1.7.10` is to add `CLOUDNS_TTL` env variable with a valid value and it should just work without any issues.

**Comment by traefiker** [comment]
Closed by #4800.
